doc/api/async_hooks.md

```diff
@@ -817,6 +817,7 @@ class WorkerPool extends EventEmitter {
       // current one.
       this.workers.splice(this.workers.indexOf(worker), 1);
       this.addNewWorker();
+      this.emit(kWorkerFreedEvent);
```

Comment:
Maybe it makes sense to move this to the end of the `addNewWorker()` function? That way it’s a bit more obvious that this is not a result of an error occurring, but rather of a new worker becoming available?

Reply:
You're right

Reply:
Updated